pyproject.toml

```diff
@@ -122,6 +122,8 @@ dummy-variable-rgx = "^(_+|(_+[a-zA-Z0-9_]*[a-zA-Z0-9]+?))$"
 "floris/simulation/wake_velocity/jensen.py" = ["F841"]
 "floris/simulation/wake_velocity/gauss.py" = ["F841"]
 "floris/simulation/wake_velocity/empirical_gauss.py" = ["F841"]
+# Ignore `F401` (import violations) in all `__init__.py` files, and in `path/to/file.py`.
+"__init__.py" = ["F401"]
```

Comment:
What's the background for this line?

Reply:
ruff will automatically delete imports in init if it doesn't detect their use, this stops that behavior:
https://github.com/astral-sh/ruff/issues/484

Reply:
I wonder how it related to this line: https://github.com/NREL/floris/blob/efd3a9589d30abd516856a36a4b2ca874214917f/pyproject.toml#L83. Is it ruff or isort that makes the change?

Reply:
ruff, I had hoped that that line in pyproject would handle it, but some how when you run ruff through pre-commit this can be ignored